### PR TITLE
Feature#28/adding functionalites and fix bugs with photos

### LINKED ISSUE #28: Adding functionalites and fix bugs with photos
# REQUIREMENTS
 1- Add funtions to menus 
 2- Fix bug with edit forms (in update all the previus photos was deleted)
 3- Add an option to display uploaded photos

# Acceptance Criteria
List all required acceptance criteria
- [1] Menus should be displayed with a click
- [2] When a user adds new photos, the previuos should not be deleted
- [3] When a user adds photos, he need to be able to show them

### PR DESCRIPTION
**Pull request by Frcomparan**
# Description
What did you implemented/Why did you implemented this:
What?
- Added drop down menu in desktop and mobile view
- Added function to delete (one at time)
- Fixed error in edit forms
- Added function to show the uploaded photos in Pets and Homes forms  
- Modified and added views for the users profile.

Why?
- To give more interactivity to the page
- To make easier the deletion of photos in Homes publications
- To fix a bug in the edit form, which deletes all previous photos even if we don't upload anything
- To let the user know what photos he uploads
- To have a view to show basic user information

  ## Screenshots 
### Form for news homes
create-home

### Form for edit homes
editar-home

### CODE REVIEW

File: app/javascript/functionalities.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+const showUserMenu = () => {
+  document.getElementById("session-menu").classList.toggle("show-menu");
+}
+
+const loadFile = (event) => {
+	let img = document.getElementById('output');
+  document.getElementById('add').style.display = 'none';
+	img.src = URL.createObjectURL(event.target.files[0]);
+  img.style.display = 'block'
```
Comment (YilKanda) [inline app/javascript/functionalities.js]: Check your indentation.

File: app/javascript/functionalities.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+};
+
+const loadFiles = (event) => {
+	let container = document.getElementById('upload');
+  document.getElementById('add').style.display = 'none';
+  const uploadPhotos = event.target.files;
+  if (document.getElementsByClassName('output').length > 0) {
+    removeChilds(document.getElementsByClassName('output'));
+  }
```
Comment (YilKanda) [inline app/javascript/functionalities.js]: Check your indentation

File: app/javascript/functionalities.js (hunk 1 above)
Comment (Frcomparan) [reply app/javascript/functionalities.js]: Fixed

File: app/javascript/functionalities.js (hunk 2 above)
Comment (Frcomparan) [reply app/javascript/functionalities.js]: Fixed

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by germanruzca — APPROVED** [review]
LGTM

**Pull request merged**